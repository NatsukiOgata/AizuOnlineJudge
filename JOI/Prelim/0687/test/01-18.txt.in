10
1559 1546 1562 1600 1538 1550 1515 1587 1598 1503
