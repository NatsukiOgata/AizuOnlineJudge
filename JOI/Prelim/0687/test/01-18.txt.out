4667
9291
